src/core/Button/index.stories.tsx

```diff
@@ -15,19 +15,16 @@ const actions = {
 const Demo = (props: Args): JSX.Element => {
```

Comment:
nit: could this be called `Button`?  If it would be a naming headache nvm.  If possible then storybook would show an entirely copy/paste-able example.

Reply:
Oh great call out! Totally missed that 😆 Will update now! Thank you!

Reply:
Updated!

<img width="1128" alt="Screen Shot 2022-08-23 at 2 29 41 PM" src="https://user-images.githubusercontent.com/6309723/186269874-4d75bbf3-1567-4c2f-971c-559f8b117e08.png">